lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -1079,6 +1080,10 @@ def message_from(response)
           response[:message]
         end
       end
+
+      def eligible_for_0_auth?(payment_method, options = {})
```

Comment:
I do see another gateway uses this method name, but I think we want to keep digits out of method names 
```suggestion
      def eligible_for_zero_auth?(payment_method, options = {})
```